src/private/props.re

```diff
@@ -138,9 +173,11 @@ let extendView =
           },
           accessibilityTraits,
         ),
-      "accessibilityViewIsModal": accessibilityViewIsModal,
-      "shouldRasterizeIOS": shouldRasterizeIOS,
-    },
-    moreProps,
-    serialize(responderHandlers),
-  );
+      ~accessibilityViewIsModal?,
+      ~shouldRasterizeIOS?,
+      (),
+    )
+    ->Obj.magic;
```

Comment:
Are you sure about this `Obj.magic`? I guess a comment would make sense on top of it to avoid loosing the rational behind it.